libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -18,7 +19,7 @@
 from .concurrency import ConcurrentQueries
 from .content import Kind
 from .embedding_model import EmbeddingModel
-from .link_tag import get_link_tags
+from .link_tag import LinkTag
```

Comment:
Suggest changing this to just `links` and `Link` as well. The original name (`LinkTag`) only supported using tags. But, I anticpate other kinds of links (for instance, we could use a special embedding model, and have a link if the source embeds near the target). So, that would be a `OutgoingEmbeddingLink` or something.

Reply:
OK, I'll do the same as in your PR. Whichever one we merge first.

Reply:
done